backend/python/app/graphql/queries/language_query.py

```diff
@@ -1,9 +1,12 @@
+from ...middlewares.auth import require_authorization_by_role_gql
 from ..service import services
 
 
+@require_authorization_by_role_gql({"User", "Admin"})
 def resolve_languages(root, info):
```

Comment:
Not sure if this can be admin only.

Reply:
users need the list of languages when signing up for story tests, so user+admin is fine